pkg/types/args.go

```diff
@@ -21,6 +21,39 @@ import (
 	"strings"
 )
 
+// UnmarshallableBool typedef for builtin bool
+// because builtin type's methods can't be declared
+type UnmarshallableBool bool
+
+// UnmarshalText implements the encoding.TextUnmarshaler interface.
+// Returns boolean true if the string is "1" or "[Tt]rue"
+// Returns boolean false if the string is "0" or "[Ff]alse"
+func (b *UnmarshallableBool) UnmarshalText(data []byte) error {
+	s := strings.ToLower(string(data))
+	switch {
```

Comment:
`*bool` doesn't have this functionality. When using the builtin bool type for the _IgnoreExtra_ field, unmarshaling on the reflection variable will throw a not implemented error.